test/e2e/simple_ovs_bridge.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NodeNetworkState", func() {
+	var ()
+	Context("when desiredState is configured", func() {
+		Context("with an ovs bridge up with no ports", func() {
```

Comment:
It is questionable whether this makes sense at all. OVS bridge without cannot be used for anything. Please drop the test.